content/guides/models/track/workspaces.md

```diff
@@ -61,6 +61,45 @@ Remove saved views that are no longer needed.
 ### Share a workspace view
 Share your customized workspace with your team by sharing the workspace URL directly. All users with access to the workspace project can see the saved Views of that workspace.
 
+## Workspace templates
+Use _workspace templates_ to quickly create new workspaces using an existing workspace's settings to new workspaces. Currently, a workspace template can include line plot settings.
+
+### Default workspace settings
+After you configure a workspace template, you can't delete it or stop using it. Instead, you can update it to use the default settings:
+
+| Setting | Default |
+|-------|----------
+| X axis             | Step |
+| Smoothing type     | Time weight EMA |
+| Smoothing weight   | 0 |
+| Max runs           | 10 |
+| Grouping in charts | on |
+| Group aggregation  | Mean |
+
+### Create a workspace template
+1. Open or create a workspace that has the [line plot settings]({{< relref "/guides/app/features/panels/line-plot/#all-line-plots-in-a-workspace" >}}) you want to templatize.
+1. Save the settings as a workspace template.
+
+  1. Click the workspace's `...` action menu at the top of the page near the **Undo** and **Redo** arrow icons.
+  1. Click **Save personal workspace template**.
+  1. Review the line plot settings that the template will use, then click **Save**.
+
+### Update a workspace template
+After you create a workspace template, follow these steps to update it:
+
+1. Open any workspace that uses the existing template.
```

Comment:
you can really just go to or open any workspace... you don't need to open one that uses and existing template

Reply:
This is confusing. How can I know if a workspace is currently using the template or not?

Reply:
hmmm why do you think that is important to the user?

you can change the settings or the workspace at anytime... this is really just trying to give you a better default experience.

Reply:
I think I got my mental map straightened out in the interim. Is there more I need to correct?